app/scripts/modules/core/src/serverGroup/serverGroup.dataSource.ts

```diff
@@ -41,10 +42,11 @@ module(SERVER_GROUP_DATA_SOURCE, [
     applicationDataSourceRegistry.registerDataSource({
       key: 'serverGroups',
       label: 'Clusters',
+      category: INFRASTRUCTURE_KEY,
       sref: '.insight.clusters',
       optional: true,
       primary: true,
-      icon: 'th-large',
+      icon: 'fas fa-xs fa-fixed fa-th-large',
```

Comment:
oh my

Reply:
I feel like I'm getting a strong signal that people really like the four-class icon.